components/software_sensor_iso_supported.mdx

```diff
@@ -0,0 +1,79 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+This sensor is included in the Clearpath Robotics operating system that is installed on all our robots.
+This allows you to add or remove the sensor from your robot's software description by changing the environment variables in `/etc/ros/setup.bash`.
+
+#### Refer to your robot's README file for more details:
+
+<Tabs groupId="ros_versions">
+  <TabItem
+    value="husky"
+    label="Husky"
+    default
+  >
+    <ul>
+      <li>
+        <a href="https://github.com/husky/husky/blob/noetic-devel/husky_description/README.md">Husky README</a>
+      </li>
+    </ul>
+  </TabItem>
+  <TabItem
+    value="jackal"
+    label="Jackal"
+  >
+    <ul>
+      <li>
+        <a href="https://github.com/jackal/jackal/blob/noetic-devel/jackal_description/README.md">Jackal README</a>
+      </li>
+    </ul>
+  </TabItem>
+  <TabItem
+    value="dingo"
+    label="Dingo"
+  >
+    <ul>
+      <li>
+        <a href="https://github.com/dingo-cpr/dingo/blob/melodic-devel/dingo_description/README.md">Dingo README</a>
+      </li>
+    </ul>
+  </TabItem>
+</Tabs>
+
+<br />
+
+#### Your robot's Tutorials page describes how the operating system is installed and configured:
```

Comment:
Might want to consider "system image" instead of "operating system", depending on what change you make to resolve the first comment above.

Reply:
I changed this to 'Clearpath Robotics System Image'.